tests/src/test/scala/cats/tests/KleisliSuite.scala

```diff
@@ -26,6 +25,8 @@ class KleisliSuite extends CatsSuite {
   implicit val iso = SemigroupalTests.Isomorphisms.invariant[Kleisli[Option, Int, ?]]
   implicit val iso2 = SemigroupalTests.Isomorphisms.invariant[Reader[Int, ?]]
 
+  implicit def cogenKlei[F[_], A, T](implicit cogen: Cogen[A => F[T]]): Cogen[Kleisli[F, A, T]] = cogen.contramap(_.run)
```

Comment:
a better place for this might be laws.discipline.arbitrary
https://github.com/typelevel/cats/blob/master/laws/src/main/scala/cats/laws/discipline/Arbitrary.scala

Reply:
Done 👍